### DISCUSSION TITLE
Is it possible to add NXP S32R45 board to zephyr RTOS

### DISCUSSION THREAD

**Discussion by Franky45** [zephyrproject-rtos/zephyr#77885] [Q&A] [answered]
Hi,

I have question like, is it possible to add NXP S32R45 board to Zephyr.
if yes how to do that let me know if there are any references for similar kind of hardware, below is the hardware link
Hardware Link -  [https://www.nxp.com/products/processors-and-microcontrollers/s32-automotive-platform/s32r-radar-processing/s32r45-evaluation-board:S32R45-EVB](url)
Thank you.

**Comment by manuargue** [comment, marked as answer]
Hello @Franky45,

thanks for your interest on Zephyr and NXP parts. At the moment there are no plans to upstream support for S32R45. If you have a use-case, I would recommend you to forward this question to the NXP community support, https://community.nxp.com/.

Best regards,
Manuel

**Reply by Franky45** [reply]
Thanks for sharing the information and suggesting further steps, Manuel. I will go ahead with that. 

Thank you,
Chaithanya
